include/seqan3/range/view/take.hpp

```diff
@@ -403,6 +406,18 @@ class view_take : public ranges::view_interface<view_take<urng_t, exactly, or_th
         ranges::copy(begin(), end(), ranges::back_inserter(ret));
         return ret;
     }
+
+    //!\overload
+    template <sequence_container_concept container_t>
+    operator container_t() const
+    //!\cond
+        requires const_iterable_concept<urng_t> && std::CommonReference<reference_t<container_t>, reference>
```

Comment:
maybe switch the arguments, s.t. it is clearer that both versions require the same, except that the const version needs `const_iterable_concept`